GVRf/Framework/framework/src/main/jni/vulkan/vulkanCore.cpp

```diff
@@ -433,6 +435,28 @@ namespace gvr {
             mRenderTexture[i] = new VkRenderTexture(width,height);
             swapChainCmdBuffer[i] = new VkCommandBuffer();
         }
+
+    }
+
+    void VulkanCore::InitPostEffectChain(){
```

Comment:
do you need separate functions for creating chain of cmd buffers? make it parameterized.